src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java

```diff
@@ -235,8 +240,17 @@ ByteArrayOutputStream createZipFile(String[] listNames, MemDataSource dataSource
         return baos;
     }
 
-    Collection<String> getAvailableFormat() {
-        return Exporters.getFormats();
+    Map<String, ExportFormatMeta> getAvailableFormat() {
+        Collection<String> formatsIds = Exporters.getFormats();
+        Map<String, ExportFormatMeta> ret = formatsIds.stream().map(formatId -> {
+            Exporter exporter = Exporters.getExporter(formatId);
+            List<ParamMeta> paramsMeta = exporter.getParameters()
+                .stream().map(pp -> new ParamMeta(pp.getName(), pp.getType(), pp.getDescription(), pp.getDefaultValue()))
+                .collect(Collectors.toList());
+            return Pair.of(formatId, new ExportFormatMeta(formatId, paramsMeta));
+
```

Comment:
remove line ?

Reply:
As you wish, tough I think this (single) empty line brought some comfort in reading.